aspnetcore/blazor/components/layouts.md

```diff
@@ -367,6 +367,61 @@ The following rendered HTML markup is produced by the preceding nested layout. E
 
 When routable components are integrated into a Razor Pages app, the app's shared layout can be used with the components. For more information, see <xref:blazor/components/prerendering-and-integration>.
 
+:::moniker range=">= aspnetcore-8.0"
+
+## Layout sections
+
+To control the content in layouts from Razor components, Blazor supports *sections* using the following built-in components:
+
+* `SectionOutlet`: Renders content provided by `SectionContent` components with matching `SectionId` or `SectionName` arguments. A `SectionOutlet` component can render the content of several `SectionContent` components.
+
+* `SectionContent`: Provides content as a <xref:Microsoft.AspNetCore.Components.RenderFragment> to `SectionOutlet` components with a matching `SectionId` or `SectionName`. Only one `SectionOutlet` component can render the content of a `SectionContent` instance.
```

Comment:
> Only one `SectionOutlet` component can render the content of a `SectionContent` instance.

This statement is true but better to say in the description for `SectionOutlet` that there cannot be two or more  `SectionOutlet`s with the same `SectionName` or `SectionId`. That will trow InvalidOperationException.